contribute/backend/package-hierarchy.md

```diff
@@ -9,56 +9,55 @@ inspiration for the way we organize packages.
 
 ## Principles of how to structure a service in Grafana
 
-[](services.md)
+To learn how to structure a Grafana service, refer to [our documentation](services.md).
 
 ### Domain types and interfaces should be in local "root" packages
 
-Let's say you're creating a _tea pot_ service, place everything another
+Let's say you're creating a "tea pot" service. You should place everything another
 service needs to interact with the tea pot service in
-_pkg/services/teapot_, choosing a name according to
-[Go's package naming conventions](https://go.dev/blog/package-names).
+`pkg/services/teapot`. Choose a name according to the
+[Go package naming conventions](https://go.dev/blog/package-names).
 
-Typically, you'd have one or more interfaces that your service provides
-in the root package along with any types, errors, and other constants
-that makes sense for another service interacting with this service to
+Typically, you have one or more interfaces that your service provides
+in the root package. Also, you should have any types, errors, and other constants
+that makes sense for another service to interact with the tea pot service to
 use.
 
 Avoid depending on other services when structuring the root package to
 reduce the risk of running into circular dependencies.
 
 ### Sub-packages should depend on roots, not the other way around
 
-Small-to-medium sized packages should be able to have only a single
+Small to medium-sized packages should be able to have only a single
 sub-package containing the implementation of the service. By moving the
 implementation into a separate package we reduce the risk of triggering
-circular dependencies (in Go, circular dependencies are evaluated per
-package and this structure logically moves it to be per type or function
-declaration).
+circular dependencies.
 
-Large packages may need utilize multiple sub-packages at the discretion
+> **Note:** In Go, circular dependencies are evaluated per
```

Comment:
not sure if lines 37 and 38 will be included in the note formatting

Reply:
my VS Code preview shows it included. to be safe, I also added semantic line breaks.